src/vsr/client_replies.zig

```diff
@@ -55,6 +61,9 @@ pub fn ClientRepliesType(comptime Storage: type) type {
         /// Track which slots have a write currently in progress.
         writing: std.StaticBitSet(constants.clients_max) =
             std.StaticBitSet(constants.clients_max).initEmpty(),
+        /// Track which slots hold a corrupt reply.
+        faulty: std.StaticBitSet(constants.clients_max) =
```

Comment:
Is it `faulty` or just `missing`?

With state-sync, I think it would be normal for replies to be missing without necessary being faulty?

Although it is faulty today, so :shrug:

Reply:
I'll update the comment to
```
        /// Track which slots hold a corrupt reply, or are otherwise missing the reply
        /// that ClientSessions believes they should hold.
```
to clarify.

I think `faulty` is a decent name even with state sync, and is more descriptive in the corruption case.
After state sync we can pretend that all of the entries are corrupt and then repair them.